examples/configure-ingress-secrets-manager/main.tf

```diff
@@ -0,0 +1,103 @@
+# This example template is for informational purposes only and is not intended for production use
+
```

Comment:
Couple of comments here

1) Examples folder should have only modules invocation. 
All resource & data source definitions should be captured as part of modules directory. 

2) Provider block should be captured as a separate file called  versions.tf

3) Have we written the terratest case for secret manager integration
Like the way we have written terratest case to integrate observability instances to cluster 
Reference 
https://github.com/terraform-ibm-modules/terraform-ibm-cluster/tree/master/examples/classic-kubernetes-e2e